spec/lib/gateway_spec.rb

```diff
@@ -179,6 +179,20 @@
       @gateway.order_status_changed(@order)
     end
 
+    it "tries to make request via http if callback_url is invalid" do
+      StraightServer::GatewayModule::CALLBACK_URL_ATTEMPT_TIMEFRAME = 1
+      uri =
+        '/?with=params&order_id=1&amount=10&amount_in_btc=0.0000001&amount_paid_in_btc=0.0&status=1&' \
```

Comment:
in each spec you have same params like `amount`, `order_id`, etc.
May be would be better to create hash with params and write small helper like `#to_param` or `#to_query`. 
More clean in this case.
http://apidock.com/rails/Hash/to_query